### ISSUE TITLE
Make Kappapp.app available on brew.sh macOS package manager

### ISSUE THREAD

**Issue by yvan-sraka** [Kappa-Dev/KaSim#557]
Hey!

I wondered about having `Kappapp.app` installable with just one line of:

```
% brew cask install kappapp
==> Satisfying dependencies
==> Downloading https://github.com/Kappa-Dev/KaSim/releases/download/v4.0/Kappapp.app.zip
==> Verifying SHA-256 checksum for Cask 'kappapp'.
==> Installing Cask kappapp
==> Moving App 'Kappapp.app' to '/Applications/Kappapp.app'.
==> Linking Binary 'KaDE' to '/usr/local/bin/KaDE'.
==> Linking Binary 'KaSa' to '/usr/local/bin/KaSa'.
==> Linking Binary 'KaSaAgent' to '/usr/local/bin/KaSaAgent'.
==> Linking Binary 'KaSim' to '/usr/local/bin/KaSim'.
==> Linking Binary 'KaSimAgent' to '/usr/local/bin/KaSimAgent'.
==> Linking Binary 'KaStor' to '/usr/local/bin/KaStor'.
🍺  kappapp was successfully installed!
```

This could be done using [Homebrew cask](https://github.com/Homebrew/homebrew-cask) (which became the canonical solution for most of macOS users).

I follow [this tutorial](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md), and write a cask we could submit with [this PR](https://github.com/Homebrew/homebrew-cask/compare/master...yvan-sraka:kappapp.app?expand=1)!

What do you think about it?

**Comment by feret** [comment]
Sounds great. Let us wait for Pirbo's return.

**Comment by yvan-sraka** [comment]
@pirbo what do you think about it? :)

**Comment by pirbo** [comment]
There are 2 levels of answers:
- can you submit kappapp to home-brew-cash ? Do I agree with your MR? Yes that looks great and yes, I don't see any missing piece.
- should home-brew-cash become a release place "officially supported by Kappa-Devs"? I'm not sure. I find already heavy to maintain publication of binaries on Github, opam packaging and pip packaging. I don't really want to have to remember about more places.... But if you find this one super critical...

**Comment by yvan-sraka** [comment]
Yes, I got your point, I don't think targeting macOS Homebrew is super critical!

We can reconsider it later if we automate our publishing process.
